extensions/amp-story-auto-ads/0.1/test/test-story-ad-localization-strings.js

```diff
@@ -0,0 +1,185 @@
+import LocalizedStringmr from '../_locales/mr.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringms from '../_locales/ms.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringmy from '../_locales/my.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringne from '../_locales/ne.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringnl from '../_locales/nl.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringno from '../_locales/no.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringpa from '../_locales/pa.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringpl from '../_locales/pl.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringptBR from '../_locales/pt-BR.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringptPT from '../_locales/pt-PT.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringro from '../_locales/ro.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringru from '../_locales/ru.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringsi from '../_locales/si.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringsk from '../_locales/sk.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringsl from '../_locales/sl.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringsq from '../_locales/sq.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringsr from '../_locales/sr.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringsv from '../_locales/sv.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringsw from '../_locales/sw.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringta from '../_locales/ta.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringte from '../_locales/te.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringth from '../_locales/th.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringtr from '../_locales/tr.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringuk from '../_locales/uk.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringur from '../_locales/ur.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringvi from '../_locales/vi.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringzhCN from '../_locales/zh-CN.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringzhTW from '../_locales/zh-TW.json' assert {type: 'json'}; // lgtm[js/syntax-error]
+import LocalizedStringzu from '../_locales/zu.json' assert {type: 'json'}; // lgtm[js/syntax-error]
```

Comment:
Seeing all these makes me wonder if it could make sense to have `/_locales/index.js` import all the json files and export an object/map containing all of them. Not sure if there are any performance/compilation implications to that approach, but it could make bits like like a good bit easier to do

Reply:
Yeah good call. As a follow up I think we could also register the bundles in the localization service using the exported array in here:
https://github.com/ampproject/amphtml/blob/5f27413e2a67897fae22892214238ec45dfa7065/extensions/amp-story/1.0/amp-story.js#L369-L391

and here

https://github.com/ampproject/amphtml/blob/5f27413e2a67897fae22892214238ec45dfa7065/extensions/amp-story-auto-ads/0.1/story-ad-localization.js#L126-L149

but we would need a reliable way of using the correct language string codes, since it's what we read from the document's `lang="string"` to make the match in the localization service with the corresponding language. /cc @gmajoulet